debian/rules

```diff
@@ -66,61 +21,9 @@ override_dh_auto_install:
 
 	touch debian/tmp/etc/machine-id
 
-	# Include all needed systemd bits
-	set -e;				\
-	for f in $$(dpkg -L systemd |	\
-	grep -E '(^/lib/systemd/system|/modprobe\.d/|/sysctl\.d/|/rules\.d/|/tmpfiles\.d/|/bin/)')	\
-	$$(dpkg -L systemd-sysv | grep sbin/)			\
-	/var/lib/systemd/					\
-	$$(dpkg -L udev | grep -E '(bin/|lib/|rules\.d/)')	\
-		;						\
-	do							\
-		LD_PRELOAD=					\
-			/usr/lib/dracut/dracut-install		\
-			-D $(CURDIR)/debian/tmp --ldd $$f;	\
-	done
-	# But we don't won't to automount the rootfs
-	rm -f --verbose debian/tmp/usr/lib/systemd/system-generators/systemd-gpt-auto-generator
-	# or this (binfmt_misc module is not in the initramfs)
-	rm debian/tmp/usr/lib/systemd/system/sysinit.target.wants/proc-sys-fs-binfmt_misc.automount
-	# Remove all pcrphase related bits for the moment (services and binary)
-	find debian/tmp/ -type f,l -name \*systemd-pcrphase\* -delete
-
-	# This hack should be removed with #113
-	sed -i '/^After=/{;s, *plymouth-start[.]service *, ,;/^After= *$$/d;}' debian/tmp/usr/lib/systemd/system/systemd-ask-password-*
-
-	# Generate hw database (/usr/lib/udev/hwdb.bin) for udev and
-	# remove redundant definitions after that.
-	systemd-hwdb --root debian/tmp update --usr --strict
-	rm -rf debian/tmp/usr/lib/udev/hwdb.d
-
-	# Use busybox shell instead of dash as in the past
-	rm debian/tmp/usr/bin/sh debian/tmp/usr/bin/dash
-	/usr/lib/dracut/dracut-install -D $(CURDIR)/debian/tmp --ldd /usr/lib/initramfs-tools/bin/busybox usr/bin/busybox
-	set -e; \
-	for alias in `debian/tmp/usr/bin/busybox --list-long | grep -v -e busybox -e reboot -e mount -e umount`; do \
-		ln -v -s busybox debian/tmp/usr/bin/$$alias; \
-	done
-
-	LD_PRELOAD= /usr/lib/dracut/dracut-install -D $(CURDIR)/debian/tmp --ldd $(INSTALL_FILES_FROM_HOST)
-	dpkg -L dmsetup | grep rules.d | xargs -L1 /usr/lib/dracut/dracut-install -D $(CURDIR)/debian/tmp --ldd
-	ln -v -s e2fsck debian/tmp/usr/sbin/fsck.ext4
-
-	set -e; \
-	for e in $$(find debian/tmp -type f -executable) \
-		/usr/lib/$(DEB_HOST_MULTIARCH)/plymouth/*.so \
```

Comment:
It seems we had a bug here where would install all dependencies of `label.so`. Now we do not. That is good. :+1:

Reply:
Correct, see also my comment to Dimitri.